packages/atomic/src/components/insight/user-actions/atomic-insight-user-action/atomic-insight-user-action.tsx

```diff
@@ -0,0 +1,100 @@
+import {Component, h, Prop, State} from '@stencil/core';
+import BookmarkIcon from '../../../../images/bookmark.svg';
+import DocumentIcon from '../../../../images/document.svg';
+import PointIcon from '../../../../images/point.svg';
+import QuickviewIcon from '../../../../images/quickview.svg';
+import SearchIcon from '../../../../images/search.svg';
+import {parseTimestampToDateDetails} from '../../../../utils/date-utils';
+import {InitializeBindings} from '../../../../utils/initialization-utils';
+import {InsightBindings} from '../../atomic-insight-interface/atomic-insight-interface';
+import {UserActionType} from '../atomic-insight-user-action-session/atomic-insight-user-actions-session';
+
+/**
+ * @internal
+ * The `AtomicInsightUserAction` component displays a single user action event in the user action timeline.
+ * @category Insight Panel
+ * @example
+ * <atomic-insight-user-action type="view" actionTitle="atlas data lake" timestamp="1723035731" origin="mySearchHub"></atomic-user-action>
+ */
+@Component({
+  tag: 'atomic-insight-user-action',
+  styleUrl: 'atomic-insight-user-action.pcss',
+  shadow: true,
+})
+export class AtomicInsightUserAction {
+  @InitializeBindings() public bindings!: InsightBindings;
+  @State() public error!: Error;
+
+  @Prop({mutable: true}) public type!: UserActionType;
+  @Prop({mutable: true}) public actionTitle!: string;
+  @Prop({mutable: true}) public timestamp!: number;
+  @Prop({mutable: true}) public origin: string = '';
+
+  icons = {
+    SEARCH: SearchIcon,
+    CLICK: DocumentIcon,
+    VIEW: QuickviewIcon,
+    CUSTOM: PointIcon,
+    TICKET_CREATION: BookmarkIcon,
+  };
+
+  renderActionTimestamp() {
+    const {hours, minutes} = parseTimestampToDateDetails(this.timestamp);
+
+    const formattedHours = String(hours).padStart(2, '0');
+    const formattedMinutes = String(minutes).padStart(2, '0');
+
+    return `${formattedHours}:${formattedMinutes}`;
+  }
+
+  renderIcon() {
+    const iconClasses = ['w-3', 'h-3'];
+    if (['CLICK', 'VIEW'].includes(this.type)) {
+      iconClasses.push('text-primary');
+    }
+
+    return (
+      <atomic-icon
+        icon={this.icons[this.type]}
+        class={iconClasses.join(' ')}
+      ></atomic-icon>
+    );
+  }
+
+  renderActionTitle() {
+    if (this.type === 'TICKET_CREATION') {
+      return (
+        <div class="font-semibold">
+          {this.bindings.i18n.t('ticket-created')}
+        </div>
+      );
+    }
+
+    const actionTitleClasses = [];
+    if (['CLICK', 'VIEW'].includes(this.type)) {
```

Comment:
Cause these are the ones that are displayed in blue, it's a color thing.

Reply:
They were displayed in blue in the mockups because they were links to the "result" they were representing.
for a view event, we have the url of the page viewed, for the click events, we had the clickableUri of the document that was clicked.

That's why they were blue. I think it's okay to have them all black for now until we decide to include to make the subsequent call to the Search API to fetch the clicked documents.